kani-compiler/src/kani_middle/transform/kani_intrinsics.rs

```diff
@@ -606,6 +618,24 @@ impl IntrinsicGeneratorPass {
             Place::from(RETURN_LOCAL),
         );
     }
+
+    fn check_float_to_int_in_range_valid_types(tcx: TyCtxt, instance: Instance) {
```

Comment:
See my comment on float.rs, I don't think we need this.

Reply:
Removed.